webui/react/src/components/SplitPane.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import css from './SplitPane.module.scss';
+
+interface Props {
+  children: [React.ReactElement, React.ReactElement];
+  initialWidth?: number;
+  onChange?: (width: number) => void;
+  open?: boolean;
+}
+
+const SplitPane: React.FC<Props> = ({ children, initialWidth, onChange, open = true }: Props) => {
+  const [isDragging, setIsDragging] = useState(false);
+  const [width, setWidth] = useState(initialWidth ?? 400);
```

Comment:
Are we handling initialWidth like this because setting the initialWidth as 0 will be overwritten if we handled it like the other default prop?
```suggestion
const SplitPane: React.FC<Props> = ({ children, initialWidth = 400, onChange, open = true }: Props) => {
  const [isDragging, setIsDragging] = useState(false);
  const [width, setWidth] = useState(initialWidth);
```

Reply:
No, I just did it like that as a placeholder before I was actually using the props and forgot to change it.